src/test/test-seccomp.c

```diff
@@ -41,6 +42,10 @@
 #  define SECCOMP_RESTRICT_ADDRESS_FAMILIES_BROKEN 0
 #endif
 
+static bool have_seccomp_privs(void) {
```

Comment:
Maybe it is better to rename and move this in test-util.[ch]? Then, we can reuse this in the next commit.

Reply:
hmm, so i thought about this, but i think the checks will end up being highly test specific. i.e. in this PR we already look for CAP_SYS_ADMIN in one place and CAP_NET_RAW in another. And in the seccomp case we could tweak this further if we wanted by also checking NNP.

Hence I figured: let's encode the perms in the individual tests, but not share them between tests, since they shouldn't be opaque but explicit, since different for each, and should be adjusted locally when needed.

Reply:
OK. That makes sense to me. Thank you fir the consideration.